static/js/github-create-branch.js

```diff
@@ -31,17 +31,35 @@ const toggleSubmitDisabled = (bool) => {
 }
 
 $(document).ready(() => {
+	// Fetching the list of default repos
+	const defaultRepos = $(".default-repos").map((_, option) => ({
+		id: $(option).html(),
+		text: $(option).html()
+	})).toArray();
+
```

Comment:
Is it worth to create a seperate endpoint that will return default repo's?  Then we don't need this workaround.

Reply:
I don't like this idea of separate endpoint just to get the default repos, as it adds loading time.
Thoughts @joshkay10

Reply:
This will improve the inital rendering time of `create-branch` page. Moreover, we can leverage repo's search endpoint. That will return default repo's if search string is empty.

Reply:
This ones worth a chat at the catchup.

I like the repos being available and not doing another request from here.

Initial rendering speed is a good call, we might have a couple more calls on the backend that we could batch anyway so could get some time back there.

As for the above work around.... dunno, could pass the data in, in a state that it doesn't need to be transformed but do we gain much?